ice-setup/config/definitions/ice-setup.yml

```diff
@@ -0,0 +1,55 @@
+- job:
+    axes:
+    - axis:
+        name: Arch
+        type: label-expression
+        values:
+        - x86_64
+    - axis:
+        name: Dist
+        type: label-expression
+        values:
+        - trusty
+        - precise
+        - centos
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell:
+        !include-raw ../../build/build
+    concurrent: false
+    description: Build ice-setup as a package for Red Hat Ceph, non-Red Hat releases
+    disabled: false
+    execution-strategy:
+      run-sequentially: false
+    keep-dependencies: false
+    name: ice-setup
+    parameters:
+    - string:
+        default: master
+        description: ''
+        name: BRANCH
+    project-type: matrix
+    properties:
+    - github:
+        url: http://github.com/ceph/ice-tools/
```

Comment:
great catch. it was wrong in the original job on jenkins-calamari. I'll fix it here.